examples/track_text.py

```diff
@@ -0,0 +1,4 @@
+import aim
+from aim import track
+
+track(aim.text, "document_name", "text")
```

Comment:
Lets track values inside a for loop to demonstrate that values are sequentially appended to the document `document_name`